tests/languages/node_test.py

```diff
@@ -113,8 +113,8 @@ def test_installs_without_links_outside_env(tmpdir):
         assert cmd_output('foo')[1] == 'success!\n'
 
 
-def _make_hello_world(tmp_path):
-    package_json = '''\
+def _make_hello_world(tmp_path, package_json=None):
+    package_json = package_json or '''\
```

Comment:
rather than overload this function -- simply call it and then overwrite the output afterwards

Reply:
ok, if you prefer it that way, will do once I resolve the breakage